core/renderers/common/constants.js

```diff
@@ -241,6 +241,18 @@ Blockly.blockRendering.ConstantProvider = function() {
    */
   this.JAGGED_TEETH_WIDTH = 6;
 
+  /**
```

Comment:
In order to make it easier to merge things in the future, why not add this in the pxt renderer? 
You can add these properties in the constructor of https://github.com/microsoft/pxt-blockly/blob/develop/core/renderers/pxt/constants.js
And then override init to call super and make the ellipses.